extensions/cornerstone/src/types/CornerstoneServices.ts

```diff
@@ -5,14 +5,26 @@ import SegmentationService from '../services/SegmentationService';
 import CornerstoneCacheService from '../services/CornerstoneCacheService';
 import CornerstoneViewportService from '../services/ViewportService/CornerstoneViewportService';
 import ViewportActionCornersService from '../services/ViewportActionCornersService/ViewportActionCornersService';
+import ColorbarService from '../services/ColorbarService';
 
-interface CornerstoneServices extends Types.Services {
-  cornerstoneViewportService: CornerstoneViewportService;
-  toolGroupService: ToolGroupService;
-  syncGroupService: SyncGroupService;
-  segmentationService: SegmentationService;
-  cornerstoneCacheService: CornerstoneCacheService;
-  viewportActionCornersService: ViewportActionCornersService;
+declare global {
+  interface CornerstoneServices extends Types.Services {
+    cornerstoneViewportService: CornerstoneViewportService;
+    toolGroupService: ToolGroupService;
+    syncGroupService: SyncGroupService;
+    segmentationService: SegmentationService;
+    cornerstoneCacheService: CornerstoneCacheService;
+    viewportActionCornersService: ViewportActionCornersService;
+    colorbarService: ColorbarService;
+  }
+
+  type cornerstoneViewportService = CornerstoneViewportService;
```

Comment:
I think this should be `import type CornerstoneViewportServiceType from '../....` and then assign here